### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by jakehockey10** [ruby-debug/ruby-debug-ide#174]
When I run the command rdebug-ide, I get a `Fatal exception in DebugThread loop: Permission denied - bind(2) for 'localhost' port 1234`.  I have tried other variations of host and port but always get a Permission denied error.  I'm attempting to use vscode with a rails project inside of the ubuntu subsystem.

**Comment by kwebos** [comment]
@jakehockey10 did you manage to solve it? i'm having the same problem...

**Comment by jakehockey10** [comment]
Nope. Really wanted vscode to work for me (I love it) but I had to go back to vim and occasionally rubymine when I feel like I need it

**Comment by kwebos** [comment]
I've uninstalled to a lower version of both ruby-debug-ide and debase and seems to be working now, but I agree, it feels like vscode and ruby are not the best fit...
